lib/YaST/Firewall/FirewallController.pm

```diff
@@ -0,0 +1,179 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: The class introduces actions Firewall Settings Page.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+package YaST::Firewall::FirewallController;
+use strict;
+use warnings;
+use testapi;
```

Comment:
we don't need `testapi`.

Reply:
I need testapi to do capture screen since in Controller level i wrap some operation together let api more simple to use in test.pm.
such as line 175.